microsoft/testsuites/xdp/xdptools.py

```diff
@@ -158,10 +165,14 @@ def _install(self) -> bool:
         # Errors happen if built with multi-threads. The program may not be
         # ready for concurrent build, but our make tool use multi-thread by
         # default. So set thread count to 1.
+        if arch == "aarch64":
+            update_envs = {"C_INCLUDE_PATH": "/usr/include/aarch64-linux-gnu/"}
```

Comment:
I thought it should be right, if the ARCH is right. Is it?

Reply:
no, specify arch doesn't work